CMakeLists.txt

```diff
@@ -1373,23 +1373,19 @@ if(UNIX AND NOT APPLE)
   if (INSTALL_USER_UDEV_RULES)
     install(
       FILES
-        "${CMAKE_CURRENT_SOURCE_DIR}/res/linux/mixxx-usb-uaccess.rules"
+        "${CMAKE_CURRENT_SOURCE_DIR}/packaging/linux/69-mixxx-usb-uaccess.rules"
       DESTINATION
-        "${MIXXX_INSTALL_DATADIR}/udev/rules.d"
+        "${CMAKE_INSTALL_LIBDIR}/udev/rules.d"
```

Comment:
By default this is "/usr/local/lib/udev/rules.d" which is wrong at least for Ubuntu

Reply:
So what is the problem? It did not install to the right place before either if the prefix was /usr/local

Reply:
Before it was installed as data to the standard arch directory under a mixxx folder. I did not consider this as clutter.
Please refer to the original decision process. https://github.com/mixxxdj/mixxx/pull/3458

Reply:
The original decision was bad, as packagers have told us. Please listen to them.

Reply:
It makes no difference whether it is installed to one wrong location or another. Neither has an effect.

Reply:
It helps nobody to install a udev rule file to a place it is certain to not have an effect. That is the present situation. This PR fixes that. There's nothing more to it.

Reply:
Note that Ubuntu _will_ install it to the correct place if it's packaging sets `-DCMAKE_INSTALL_PREFIX=/`. If it doesn't right now, that's wrong anyhow.

Reply:
Unfortunately it does not, see:  
https://github.com/Debian/debhelper/blob/eb6e510ef52430c6edb96b28879fa4f64fbcafdb/Debian/Debhelper/Buildsystem/autoconf.pm#L34

Reply:
That's autoconf and not relevant for a CMake project. Besides, you should be able to overwrite whatever defaults it sets as a packager.

Reply:
Oh right, wrong link:
https://github.com/Debian/debhelper/blob/eb6e510ef52430c6edb96b28879fa4f64fbcafdb/Debian/Debhelper/Buildsystem/cmake.pm#L15

> Besides, you should be able to overwrite whatever defaults it sets as a packager.

Yes, but the point of the whole PR is to avoid manual steps, right?

Reply:
I find configuring CMake correctly by adding an argument to the command you're calling any way better than executing an entirely new command :wink: 

But it seems Ubuntu installs things in places I didn't expect, so I guess the pkg-config method is the way to go. Closing this PR for now.